db/seeds.rb

```diff
@@ -5,3 +5,10 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+Message.create([
+  { message: 'Hello' },
+  { message: 'How are you?' },
+  { message: 'What is your name?' },
+  { message: 'Where are you from?' },
+  { message: 'Nice too meet you' }
```

Comment:
- [ ] It would be better if we fix this part of our code, it gives us error when we try seeding the data, because the attribute `message` is not found in our schema but rather `text` so kindly change the `message` attribute to `text` for the database to seed successfully

Reply:
I will fix it.